shell/scripts/test-plugins-build.sh

```diff
@@ -2,22 +2,132 @@
 
 set -e
 
-echo "Checking plugin build"
+CYAN="\033[96m"
+YELLOW="\033[93m"
+RESET="\033[0m"
+BOLD="\033[1m"
+NORMAL="\033[22m"
+
+echo -e "${BOLD}${CYAN}Extensions framework build tests${RESET}"
 
 SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 BASE_DIR="$( cd $SCRIPT_DIR && cd ../.. & pwd)"
 SHELL_DIR=$BASE_DIR/shell/
 
-echo ${SCRIPT_DIR}
+# ---- Options ----------------------------------------------------------------------------------------------------
+
+KEEP_FOLDER=false
+KEEP_PKG=false
+SKIP_IN_TREE=false
+SKIP_SETUP=false
+SKIP_STANDALONE=false
+
+REG_USER=admin
+REG_PWD=admin
+
+usage() {
+  echo "Usage: $0 [<options>] [plugins]"
+  echo " options:"
+  echo "  -k           Keep the folder for the test application after run"
+  echo "  -e           Keep the test extension after run (in-tree tests)"
+  echo "  -i           Skip in-tree tests"
+  echo "  -s           Skip standalone tests"
+  echo "  -x           Skip test setup"
+  echo "  -u <name>    Verdaccio user name for publishing (default: admin)"
```

Comment:
I am not able to login with this flag:

```
➜  dashboard git:(extension-test-doc) ./shell/scripts/test-plugins-build.sh -u admin -p admin          
Extensions framework build tests
./shell/scripts/test-plugins-build.sh: illegal option -- u
Usage: ./shell/scripts/test-plugins-build.sh [<options>] [plugins]
 options:
  -k           Keep the folder for the test application after run
  -e           Keep the test extension after run (in-tree tests)
  -i           Skip in-tree tests
  -s           Skip standalone tests
  -x           Skip test setup
  -u <name>    Verdaccio user name for publishing (default: admin)
  -p <name>    Verdaccio password for publishing
```

This comment has been added just because the help (cool thing!) is mentioning it but it did not work for me